packages/flutter_test/lib/src/_leak_tracking.dart

```diff
@@ -0,0 +1,67 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:leak_tracker/leak_tracker.dart';
+
+import 'leak_tracking.dart';
+import 'test_async_utils.dart';
+import 'widget_tester.dart';
+
+/// Unit test friendly wrapper for [withLeakTracking] with Flutter specific functionality.
+///
+/// The method will fail with test matcher error if wrapped code contains memory leaks.
+///
+/// See details in documentation for `withLeakTracking` at
+/// https://github.com/dart-lang/leak_tracker/blob/main/lib/src/orchestration.dart#withLeakTracking
```

Comment:
This link just seems to direct me to the top of the file and as such is not that helpful. Also, the content there could get out of date or the file may move. Instead, we should reproduce all relevant information here in the doc.

Reply:
Also, document what the parameters mean.

Reply:
Documented parameters.

This function is (1) internal and (2) just wraps withLeakTracking, without being concerned about its meaning. So, I do not see value in copying documentation
from leak tracker. 
I will better improve the documentation for withLeakTracking: https://github.com/dart-lang/leak_tracker/pull/45